account_move_tier_validation/views/account_move_view.xml

```diff
@@ -2,88 +2,6 @@
 <!-- Copyright <2020> PESOL <info@pesol.es>
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl) -->
 <odoo>
-    <record id="view_move_form" model="ir.ui.view">
```

Comment:
Why is this form view removed?

Reply:
I rely on the [autoconfiguration](https://github.com/OCA/server-ux/blob/16.0/base_tier_validation/models/tier_validation.py#L477) of the base module because I didn't see this adding anything the base module doesn't - did I miss something?

Reply:
You are right, I just tried on runbot and it works fine. Thanks!

What I noticed is that the "Request Validation" button is not displayed until I press the button "Save manually". It seems to me confusing, as I would expect to see the "Request Validation" button as soon as I complete to fill the invoice. What do you think?

Reply:
Yes, is a generic issue of usability that could be addressed to base module, thanks!

Reply:
@astirpe see https://github.com/OCA/server-ux/pull/640, with this also the confirm button should be hidden immediately if you fill in an invoice that will be in validation